go/.gitignore

```diff
@@ -0,0 +1 @@
+paddlecloud/paddlecloud
```

Comment:
[here](https://github.com/PaddlePaddle/cloud/blob/develop/go/paddlecloud/.gitignore#L1) is a `.gitignore` file.I think this is unnecessary.